client/src/partials/fiscal/fiscal.js

```diff
@@ -0,0 +1,105 @@
+angular.module('bhima.controllers')
+.controller('FiscalController', FiscalController);
+
+FiscalController.$inject = ['$state', 'FiscalService', '$interval', 'ModalService', 'NotifyService'];
+
+/**
+
+ */
+function FiscalController($state, fiscalService, $interval, ModalService, Notify) {
+  var vm = this;
+  var today = new Date(); 
+
+  // pagination configuration
+  /** @todo this should all be moved to a component */
+  vm.pageSize     = 5;
+  vm.currentPage  = 1;
+  vm.fiscalYears  = [];
+  vm.maxSize      = 5;
+  vm.del          = del;
+
+  vm.sort = sort;
+  vm.state = $state;
+
+  vm.sortByName = "ASC";
+  vm.sortByDateCreated = "ASC";
+  vm.sortByNumbrerOfMonth = "ASC";
+
+  // refreshFiscalYear
+  function refreshFiscalYear(){
+    return fiscalService.read(null,{ detailed : 1 })
+    .then(function (fiscalYears) {
+      vm.fiscalYears = fiscalYears;
+    });
+  }
+
+  // Get the fiscal Year By Date
+  fiscalService.fiscalYearDate({date : today})
+  .then(function (current) {
+    vm.current = current;
+  })
+  .catch(handleException);
+
+  function handleException(error) {
```

Comment:
Instead of handleException, I think this would be better used for `Notify.handleError()`.

`Notify.handleError()` exists to inform the user of a problem they cannot correct.  If there is trouble reading from the database, they should be informed.